### ISSUE TITLE
STEREO LFOs

### ISSUE THREAD

**Issue by hems** [surge-synthesizer/surge#5838]
**Is your feature request related to a problem? Please describe.**
It would be great to have "stereo phase" on the LFOS.

UVI SHADE has that implemented really nicely ( they called it "SPREAD" ) where you can make an LFO "stereo".

For instance you can assign an LFO to CUTOFF and then play with the "STEREO SPREAD" of the LFO which basically would ( as you guessed ) shift the PHASE  of the modulation between LEFT and RIGHT side of the filter.

At 180 phase when a triangle would be going up for the right side on the cut-off it would be going down on the left side.

this is a really powerful and easy way of making stereo sounds.

This example is similar to how the PHASE knob works on Ableton's `Auto Filter` as well.

**Comment by mkruselj** [comment]
Modulation in Surge is strictly mono, this would probably be a pretty big overhaul of the innards. I suggest using Vital instead. 🙂

**Comment by baconpaul** [comment]
Yeah i was more enamored of this one. I'd like to keep it open as a speculative issue. With vector valued modulators which we just added it is way more tractable than with the 19 code base.

The trick would be to have stereo-modulatable parameters. cutoff and resonance would be one such param. some of the fx ones would also i bet. So will tag it currently unscheduled but will ponder
